certora/checker/Checker.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: GNU AGPLv3
+pragma solidity ^0.8.0;
+
+import "../../lib/morpho-utils/lib/openzeppelin-contracts/contracts/utils/Strings.sol";
+import "../helpers/MerkleTreeLib.sol";
+import "../../lib/forge-std/src/Test.sol";
+import "../../lib/forge-std/src/StdJson.sol";
+
+contract Checker is Test {
+    using MerkleTreeLib for MerkleTreeLib.Tree;
+    using stdJson for string;
+
+    MerkleTreeLib.Tree public tree;
+
+    struct Leaf {
+        address addr;
+        uint256 value;
+    }
+
+    struct InternalNode {
+        address addr;
+        address left;
+        address right;
+    }
```

Comment:
A Merkle tree is a tree of hashes, so it naturally has identifiers as bytes32. I changed it on purpose to address, where the addresses of the leaves are exactly the account receiving receiving the rewards, so that we make sure that there is not twice the same address in a single tree

Reply:
Done [here](https://github.com/morpho-org/morpho-optimizers/blob/48e40e8baf0685589eaf1ed3e140806e2ca77191/certora/helpers/MerkleTreeLib.sol#L32-L33)